docs/Developers/Git_Basics.rst

```diff
@@ -539,6 +539,8 @@ commands (described below) for each commit to be rebased. These commands
 determine how individual commits will be transferred to the new base. You can
 also reorder the commit listing to change the order of the commits themselves.
 
+For more commands go to http://ohshitgit.com/
```

Comment:
1. Missing period
2. This doesn't read so nice and is not really helpful, it's just a dropped link. You should try to highlight the fact that this link is very useful if something goes wrong. It's not just a bunch of commands^^ (well in fact it is, but commands helping you to get out of a screwed-state ;) )